packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -501,6 +530,149 @@ export class ParticleGenerator {
   _destroy(): void {
     this._instanceVertexBufferBinding.buffer.destroy();
     this._primitive.destroy();
+    this.emission._destroy();
+  }
+
+  /**
+   * @internal
+   */
+  _updateBoundsSimulationLocal(bounds: BoundingBox): void {
+    const renderer = this._renderer;
+    // Get longest Lifetime
+    const maxLifetime = this.main.startLifetime._getMax();
+
+    const generatorBounds = renderer._generatorBounds;
+    if (renderer._isContainDirtyFlag(ParticleUpdateFlags.GeneratorVolume)) {
+      this._calculateGeneratorBounds(maxLifetime, generatorBounds);
+      renderer._setDirtyFlagFalse(ParticleUpdateFlags.GeneratorVolume);
+    }
+
+    const transformedBounds = this._tempBoundingBox;
```

Comment:
Why not call `_transformedBounds` directly, and store in `ParticleRenderer` with `_generatorBounds` togethor